src/lib/translations.js

```diff
@@ -67,21 +67,75 @@ export default {
 			back: 'Wstecz',
 			save: 'Zapisz i zamknij'
 		},
+		tabs: {
+			tab1: {
+				menu: 'Informacje wydawcy',
+				title: 'Lorem ipsum dolor sit amet',
+				description: '<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc fringilla nisl erat, sodales molestie tortor accumsan sed. Vivamus at massa suscipit, ornare magna quis, tincidunt ante. Sed at enim vel est aliquet pharetra ut vehicula enim. In scelerisque justo orci, ac vehicula ligula finibus blandit. Vivamus non convallis nunc. Integer vitae vestibulum est, at scelerisque diam. Morbi blandit a erat vitae fermentum. Duis pulvinar pretium hendrerit. Vestibulum ligula justo, mattis nec tortor eu, dictum tempor urna. Quisque varius finibus neque vitae pellentesque. Curabitur pharetra vitae neque porta blandit. Suspendisse commodo lacinia sapien, ac egestas neque pharetra ut. Sed in tincidunt urna, id porttitor ipsum. In neque tortor, condimentum at est sed, tempor dignissim odio. Pellentesque suscipit, quam sit amet efficitur facilisis, enim nibh rhoncus nulla, id malesuada lectus ante et justo. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos.</p>',
+			},
+			tab2: {
+				menu: 'Cele i funkcje przetwarzania danych osobowych',
+				title: '',
+				description: '',
+			}
+		},
+		publisherConsents: {
+			title: "Zgody dla wydawcy",
+			description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc fringilla nisl erat, sodales molestie tortor accumsan sed.',
+		},
+		vendorConsents: {
+			title: "Zgody dla partnerów",
+			description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc fringilla nisl erat, sodales molestie tortor accumsan sed.',
+		},
 		purposes: {
 			active: 'Aktywna',
 			inactive: 'Nieaktywna',
 			showVendors: 'Pokaż listę Zaufanych Partnerów',
 			showCustomVendors: 'Pokaż listę pozostałych Zaufanych Partnerów',
+			title: 'Cele',
```

Comment:
As I see this label is also used as heading inside second tab. I'm not sure if using the same label here will be correct because someone may want to use something else in vendors' description (e.g. "Cele przetwarzania na podstawie zgody") and at the same time different text as heading.